migrations/app/schema/20241119151019_stored_procs_for_ordering_service_items.up.sql

```diff
@@ -0,0 +1,364 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+ALTER TABLE mto_service_items
+  ALTER COLUMN id SET DEFAULT uuid_generate_v4();
+
+
+-- creating function to get address is_oconus
+CREATE OR REPLACE FUNCTION get_is_oconus(address_id UUID)
+RETURNS BOOLEAN AS $$
+DECLARE
+    is_oconus BOOLEAN;
+BEGIN
+    SELECT a.is_oconus
+    INTO is_oconus
+    FROM addresses a
+    WHERE a.id = address_id;
+
+    RETURN is_oconus;
+EXCEPTION
+    WHEN NO_DATA_FOUND THEN
+        RAISE EXCEPTION 'Address with ID % not found', address_id;
+END;
+$$ LANGUAGE plpgsql;
+
+CREATE OR REPLACE FUNCTION checkExistingServiceItem(
+    service_id UUID,
```

Comment:
Should we rename to be a more descriptive func name like `does_service_item_exist`?
```suggestion

CREATE OR REPLACE FUNCTION does_service_item_exist(
    service_id UUID,
```

Reply:
to those sound the same.. but if its clearer to you that way.. ill update

Reply:
to those sound the same.. but if its clearer to you that way.. ill update

Reply:
It sounds like it if you already know what it does, but for unfamiliar devs they may not know exactly what the "check" condition is